python/graphstorm/gpartition/dist_partition_graph.py

```diff
@@ -154,6 +158,26 @@ def main():
 
         logging.info("DGL graph building took %f sec", part_end - time.time())
 
+    if args.use_graphbolt:
+        dgl_version = importlib.metadata.version('dgl')
+        if version.parse(dgl_version) >= version.parse("2.0.0"):
+            # TODO: Use dgl.distributed.partition.gb_convert_single_dgl_partition()
```

Comment:
Why not use now? Can you also add the reason in the comment?

Reply:
It's a non-trivial implementation (we need to write pytorch.distributed/MPI-style code) that I'd rather leave for later.

Reply:
I c.